### DISCUSSION TITLE
Add additional custom databases

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco.Cloud.Issues#793] [Ideas]
### Issue description

By default on Umbraco Cloud we have a database per environment on Umbraco Cloud, but for some projects in makes sense to add addtional databases (maybe per environment?).

For example Vendr now also supports the Vendr specific tables to be stored in a different database. There are other use-cases where developers may want to store additional data in a separate database instead on on the same database as Umbraco tables.

Maybe it could also be considered from a security point of view, e.g. Alice need access to work on Candy Shop project, but only Umbraco content, while Bob need access to both Umbraco and Vendr in backoffice and databases.

Currently one would have to add custom databases external, e.g. on Azure and connect to this, but it would be useful to added this from Umbraco Cloud (which use Azure under the hood). It also wouldn't require and additional Azure account and may be simpler for developers who are not familiar with Azure.

**Comment by sajumb** [comment]
Hi @bjarnef 
We apologize for the late response. But thanks for the feedback. We have noted the feature requested and we will return with an update if we begin to consider this request.
